Lib9c/Model/State/GameConfigState.cs

```diff
@@ -130,14 +172,50 @@ public override IValue Serialize()
             {
                 values.Add((Text)"worldboss_required_interval", WorldBossRequiredInterval.Serialize());
             }
+
+            if (StakeRegularFixedRewardSheet_V2_StartBlockIndex > 0)
+            {
+                values.Add(
+                    (Text) "stake_regular_fixed_reward_sheet_v2_start_block_index",
+                    StakeRegularFixedRewardSheet_V2_StartBlockIndex.Serialize());
+            }
+
+            if (StakeRegularRewardSheet_V2_StartBlockIndex > 0)
+            {
+                values.Add(
+                    (Text) "stake_regular_reward_sheet_v2_start_block_index",
+                    StakeRegularRewardSheet_V2_StartBlockIndex.Serialize());
+            }
+
+            if (StakeRegularRewardSheet_V3_StartBlockIndex > 0)
+            {
+                values.Add(
+                    (Text) "stake_regular_reward_sheet_v3_start_block_index",
+                    StakeRegularRewardSheet_V3_StartBlockIndex.Serialize());
+            }
+
+            if (StakeRegularRewardSheet_V4_StartBlockIndex > 0)
+            {
+                values.Add(
+                    (Text) "stake_regular_reward_sheet_v4_start_block_index",
+                    StakeRegularRewardSheet_V4_StartBlockIndex.Serialize());
+            }
+
+            if (StakeRegularRewardSheet_V5_StartBlockIndex > 0)
+            {
+                values.Add(
+                    (Text) "stake_regular_reward_sheet_v5_start_block_index",
+                    StakeRegularRewardSheet_V5_StartBlockIndex.Serialize());
+            }
+
 #pragma warning disable LAA1002
             return new Dictionary(values.Union((Dictionary) base.Serialize()));
 #pragma warning restore LAA1002
         }
 
         public void Set(GameConfigSheet sheet)
         {
-            foreach (var row in sheet.OrderedList)
+            foreach (var row in sheet.OrderedList!)
```

Comment:
This seems incorrect `nullable` type remover. How can it guarantee `Sheet.OrderedList` is not null at that time?